### DISCUSSION TITLE
关于过验证码的方式

### DISCUSSION THREAD

**Discussion by Seventhdoge** [mikumifa/biliTickerBuy#303] [提问专区] [answered]
程序给出了三个过验证码方式，我想问的是“本地过验证码（Amorter提供）”这个功能真的会自动识别验证码吗，还是说最好老老实实手动过验证码？有没有实战过的朋友回答一下

**Comment by CinnabarCHU** [comment, marked as answer]
Amorter大大给的这玩意巨好用，感觉真的没必要手打……

**Reply by Seventhdoge** [reply]
OK明白了~谢谢解开我心头疑惑，虽然说还不能确定会出其他什么小插曲...前两次给我抢麻了

**Reply by Hanabiiiiii** [reply]
> Amorter大大给的这玩意巨好用，感觉真的没必要手打……

![image](https://github.com/mikumifa/biliTickerBuy/assets/122472858/806404cc-4425-44ce-96d8-7060176a3016)
zheyan这样本地测试过了是不是说明就可以用了

**Reply by Seventhdoge** [reply]
我就是这种结果，看来可以暂且安心一会了
